base/error.jl

```diff
@@ -68,6 +68,8 @@ end
 # convert dual arrays (raw bt buffer, array of GC managed values) to a single
 # array of locations
 function _reformat_bt(bt, bt2)
+    bt::Vector
+    bt2::Vector
```

Comment:
Any reason not to put `bt::Vector{Ptr{Cvoid}}, bt2::Vector{Any}` in the signature instead?

Reply:
Oops, leftovers from a Revise/Cthulhu experiment that I forgot to finalize.

Reply:
Oh, wait, no, there was a reason: `Vector` isn't defined at this point in bootstrap yet.